run_kraken.py

```diff
@@ -37,9 +39,28 @@ def main(cfg):
     logging.info("Starting kraken")
 
     # Parse and read the config
-    if os.path.isfile(cfg):
-        with open(cfg, "r") as f:
-            config = yaml.full_load(f)
+    # Initializing flag variable as 0 to test config is a file or not
+    flag = 0
+
+    try:
+        # For config as remote file
+        if urlparse(cfg):
+            # Set flag as 1 since config is remote file
+            flag = 1
+            content = requests.get(cfg)
+            texts = content.text
+            config = yaml.safe_load(texts)
+
+    except Exception:
+        # For config as local file
+        if os.path.isfile(cfg):
```

Comment:
this if statement/code block should also be under the try, want to be sure that the config's we are passing (whether by url or a local config) is a yaml file layout

Will want to exit out of kraken if an exception is thrown here